preCICE-Ubuntu-20.04/Vagrantfile

```
@@ -0,0 +1,44 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  # The Vagrant documentation recommends the bento images instead of ubuntu (more providers, open source)
+  config.vm.box = "bento/ubuntu-20.04"
+
+  # We don't want the box to automatically update every time it starts.
+  # We can instead handle updates internally, without destroying the machine.
+  config.vm.box_check_update = false
+
+  # Share an additional folder to the guest VM. The first argument is
+  # the path on the host to the actual folder. The second argument is
+  # the path on the guest to mount the folder. And the optional third
+  # argument is a set of non-required options.
+  # config.vm.synced_folder "../data", "/vagrant_data"
```

Not sure if the duplication works.

```suggestion
  # config.vm.synced_folder "../data", "/vagrant_data"
  # Disable the default folder sync
  config.vm.synced_folder ".", "/vagrant", disabled: true
  # Sync a dedicated shared folder
  config.vm.synced_folder "./shared", "/home/vagrant/Desktop/shared"
```

---

I don't see the reason to disable the `/vagrant/` part, but I like the explicit `~/Desktop/shared/`, I added it.